### DISCUSSION TITLE
Support for OpenTelemetry

### DISCUSSION THREAD

**Discussion by darshanpatil** [conductor-oss/conductor#103] [Q&A]
Is it possible to integrate conductor with Open telemetry/Prometheus for improving observability?

**Comment by Robban1980** [comment]
Is there something missing from the existing Prometheus endpoint?

**Reply by andyren0110** [reply]
e.g. we'd like to get the ability to export distributed traces and then view the trace in jaeger etc.

**Reply by tusharshaily** [reply]
Hi @andyren0110 , did you got any integration solution to trace in Jaeger UI using OpenTeleMetry ?
